RELEASE_NOTES.md

```diff
@@ -4,21 +4,51 @@ Any new bugs, requests, or missing features should be reported as [Github issues
 
 ## Improvements:
 
+### Code generation
+ * Add support for float16 encoding and decoding
+ * Add support for C++ in generated code
+ * Use (U)INT(8|16|32|64)_(MIN|MAX) macros in generated code
+ * Add the --c-code-columns option for adding newlines to c_code
+ * Refactor CBOR <-> YAML/JSON translation
+ * Allow adding file headers to generated files, e.g. for copyright.
+
+### C libraries
+ * Add support for float16 encoding and decoding
+ * Add helper functions zcbor_compare_strings() and zcbor_header_len()
+ * Rename prim/primitive to simple
+ * Add new APIs zcbor_uint_(encode/decode)
+ * Move tags enum into separate file zcbor_tags.h and add more tags from the IANA list
+ * When generating cmake files, make all paths relative to the cmake file.
+ * Use the zcbor_compare_strings() function in str_expect()
+ * Make the zcbor source files compile with -Wall and -Wconversion
+
+### Other
+ * README: Improve docs, particularly to help new users
+ * Add two samples: a hello world with no code generation, and one with code generation and script usage
+ * Port decode/encode/unit tests to new ztest API
+ * Fix fuzz tests
+
 ## Bugfixes:
+ * zcbor.py: Bring back 'zcbor --version'
+ * zcbor.py: Fix cbor bstrs so they properly check unambiguous values
+ * zcbor.py: Fix rendering of entry functions when they have no output
+ * zcbor.py: Don't flag cbor bstrs as unambiguous
```

Comment:
Can this be combined with "Fix cbor bstrs so they properly check unambiguous values" or are these 2 different bugs?

Reply:
Two different bugs